site/src/nav/DownloadButton.jsx

```diff
@@ -72,13 +82,25 @@ function DownloadButton() {
 
     const center = myMap.getCenter();
     const zoom = myMap.getZoom();
-    downloadLink.download = `overture-${zoom}-${center.lat}-${center.lng}.geojson`;
+
+    if (fileName === "") {
+      downloadLink.download = "overture.geojson";
+    } else {
+      downloadLink.download = `${fileName}.geojson`;
+    }
+
+    // downloadLink.download = `overture-${zoom}-${center.lat}-${center.lng}.geojson`;
```

Comment:
I'd prefer if we leave commented out code out of `main`.

Reply:
Definitely will do this, there is some other dead code that can be removed as well.